frontend/src/store/playerElement.ts

```diff
@@ -154,6 +180,19 @@ class PlayerElementStore {
         (sub) => sub.srcIndex === playbackManager.currentSubtitleStreamIndex
       );
 
+      const attachedFonts =
+        playbackManager.currentMediaSource?.MediaAttachments?.filter((attach) =>
+          this._isSupportedFonts(attach.MimeType || '')
```

Comment:
```suggestion
        playbackManager.currentMediaSource?.MediaAttachments?.filter((a) =>
          a.MimeType ? this._isSupportedFonts(a.MimeType) : false
```

I think this one is less obscure for a filter function